src/App.jsx

```diff
@@ -1,12 +1,76 @@
-import './App.css'
+import { Route, Routes, useLocation } from 'react-router-dom';
+import { HiMenuAlt4 } from 'react-icons/hi';
+import { useDispatch } from 'react-redux';
+import Homepage from './pages/Homepage';
+import NoMatch from './pages/NoMatch';
+import './styles/App.css';
+import Landingpage from './pages/Landingpage';
+import ReservationNew from './pages/ReservationNew';
+import Navigation from './components/Navigation';
+import { toggleNav } from './redux/features/NavbarSlice';
+import Authentication from './pages/Authentication';
+import MyReservations from './pages/MyReservations';
+import AdminPages from './pages/AdminPages';
+import AdminOutlet from './pages/AdminOutlet';
+import RemoveStudios from './pages/RemoveStudios';
+import AddStudio from './pages/AddStudio';
+import Studiodetails from './pages/StudioDetailspage';
+import ProtectedPages from './pages/ProtectedPages';
+import UserOutlet from './pages/UserOutlet';
 
 function App() {
```

Comment:
- [ ] Since your app is wired to consume the server in the cloud, I can't access the admin section to test the `delete` and `add` functionality.

Kindly, include an admin user with credentials either on your readme or your PR description so people can test the admin-exclusive functionality. 👍